src/Microsoft.AspNet.Mvc.DataAnnotations/DataTypeClientModelValidatorProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// An implementation of <see cref="IClientModelValidatorProvider"/> which provides client validators
+    /// based on the model type.
```

Comment:
In that case, as @pranavkm already suggested, should I change the name of the class to `NumberDataTypeClientModelValidatorProvider`?